### DISCUSSION TITLE
Unicorn partial using classes

### DISCUSSION THREAD

**Discussion by pfisterjonas** [django-commons/django-unicorn#548] [Ideas]
Is there a way to use unicorn partial with classes instead of keys or ids or to at least pass in an array of ids? It would make it much easier to accomodate for dom changes through javascript on the frontend (like multiple modals or popovers with dynamic content in a dashboard where setting unicorn ignore is not always an option)

**Comment by adamghill** [comment]
Targeting by `class` or an array of ids is not currently possible because the code that handles partials assumes it is a very small/focused part of the component.

This is where the component partial dom gets re-rendered server-side: https://github.com/adamghill/django-unicorn/blob/3c6391d00f60351f63279eca7c3454043667e0b7/django_unicorn/views/__init__.py#L244

This is where the JS happens: https://github.com/adamghill/django-unicorn/blob/3c6391d00f60351f63279eca7c3454043667e0b7/django_unicorn/static/unicorn/js/messageSender.js#L204-L208

_Maybe_ it'd be possible to send back multiple re-rendered partials from the server and then morph each partial client-side. If you want to create a PR to at least start he process I can take a look and help finish it up.
